textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -0,0 +1,38 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Class to take in a string and uses a dictionary of words, to split it into possible
+ * set of strings, with appropriate space delimitions. his is a resource used by the QueryRewriter
+ * operator
+ *
+ *
+ * For example - "newyorkcity" - > ["new york city", "newyorkcity"]
+ * For example - "horseshoe" -> ["hor se shoe", "hors es hoe", "horse shoe", "horses hoe", "horseshoe"]
+ */
+public class FuzzyTokenizer {
```

Comment:
Why do we need a separate FuzzyTokenizer class?  Why cannot we include it in the QueryRewriter class directly?

In addition, this PR should only include the interface and test cases, WITHOUT the implementation.  Can you remove the code related to implementation?